### ISSUE TITLE
Cannot read property 'isPromise' of undefined

### ISSUE THREAD

**Issue by stevenorell-dc** [duan602728596/gulp-terser#15]
Possibly related to Issue #14 

Getting an error:
```
TypeError in plugin "terser"
Message:
    Cannot read property 'isPromise' of undefined
```

Configuration:
```
gulp
    .src(files.js.src)
    .pipe(terser())
    .on('error', function (error) {
      console.error('Error in js.src compress task', error.toString());
    })
    .pipe(concat('admin.min.js'))
    .pipe(gulp.dest(path.js.build));
```

**Versions:**
npm@6.4.1
gulp@3.9.1
gulp-terser@1.3.0
terser@5.0.0

**Comment by jamesray** [comment]
Having the same issue, just started with this latest version.

**Comment by duan602728596** [comment]
It may be that your node version is too low. I used the `util.types.isPromise` method to determine whether it is a promise. This problem has been fixed.

**Comment by duan602728596** [comment]
It may be that you are using a low version of node and I used the `util.types.isPromise` method. Has been fixed in v1.3.2.
